jbpm/jbpm-flow/src/main/java/org/jbpm/process/core/impl/DataTransformerRegistry.java

```diff
@@ -70,18 +71,19 @@
 public class DataTransformerRegistry {
 
 	private static final Logger logger = LoggerFactory.getLogger(DataTransformerRegistry.class);
-	private static final DataTransformerRegistry INSTANCE = new DataTransformerRegistry();
+    private static final DataTransformerRegistry INSTANCE =
+        // do not create ScriptEngineManager on SVM for now
+        System.getProperty("org.graalvm.nativeimage.imagecode") == null?
+            new DataTransformerRegistry(new ScriptEngineManager().getEngineFactories()) :
```

Comment:
I would actually remove the ScriptEngineManager completely ... if needed we can add it later on

Reply:
I'm fine with that, so what about nodes that look for that (e.g. SubProcess) ? they'll just get `null` ?

Reply:
it should only be used for transform operation in data inputs and outputs and nothing else. So it should be limited to few tests affected by the removal. Don't think nodes itself care about it

Reply:
good, let's nuke it then :-P